samples/net/echo_client/prj_cc2520.conf

```diff
@@ -13,7 +13,6 @@ CONFIG_NET_IPV6=y
 CONFIG_NET_IPV6_RA_RDNSS=y
 CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=2
 
-CONFIG_NET_6LO=y
 CONFIG_NET_DEBUG_6LO=n
```

Comment:
i would remove this as well, first, it is weird to have it here when the option relate to it is being removed, and then it is set to n, which should be the default anyways.

Reply:
I left over yes, will remove